### PR TITLE
Update main to go 1.19.2

### PR DESCRIPTION
**Pull request by mpalmi**
Dropping _go_ver-1120 seemed correct, but would like verification.

### CODE REVIEW

**Comment by ccapurso** [comment]
> Not sure about deleting the 1.12 go version changelog entry, it does kinda make sense to remove it.

The change/_go* entries are a recent thing, correct? I typically look as the changelog as an additive thing so removing something seems a bit off but it doesn't seem like there is much precedent in the git log for how to handle these entries.

**Comment by stevendpclark** [comment]
> > Not sure about deleting the 1.12 go version changelog entry, it does kinda make sense to remove it.
> 
> The change/_go* entries are a recent thing, correct? I typically look as the changelog as an additive thing so removing something seems a bit off but it doesn't seem like there is much precedent in the git log for how to handle these entries.

True it does seem odd to remove the entry on main but the same file exists with a different value now on release/1.12.x now. It's unclear which one the changelog would pick up? Perhaps this needs to be added to the branching procedure to clean up these entries?

**Comment by mpalmi** [comment]
> > > Not sure about deleting the 1.12 go version changelog entry, it does kinda make sense to remove it.
> > 
> > 
> > The change/_go* entries are a recent thing, correct? I typically look as the changelog as an additive thing so removing something seems a bit off but it doesn't seem like there is much precedent in the git log for how to handle these entries.
> 
> True it does seem odd to remove the entry on main but the same file exists with a different value now on release/1.12.x now. It's unclear which one the changelog would pick up? Perhaps this needs to be added to the branching procedure to clean up these entries?

Yeah, I think that's where it got tricky for me to reason about. In theory, we'd update 1.12 and main with the change (and introduce a new changelog entry), but it seems like each branch should have its own changelog(-set) for Go versions.

I'll see if there's an outright disqualifier when running `changelog-build`. In the absence of that, we can leave the 1120 entry around until we decide on and document a process for this.

**Comment by mpalmi** [comment]
> > > > Not sure about deleting the 1.12 go version changelog entry, it does kinda make sense to remove it.
> > > 
> > > 
> > > The change/_go* entries are a recent thing, correct? I typically look as the changelog as an additive thing so removing something seems a bit off but it doesn't seem like there is much precedent in the git log for how to handle these entries.
> > 
> > 
> > True it does seem odd to remove the entry on main but the same file exists with a different value now on release/1.12.x now. It's unclear which one the changelog would pick up? Perhaps this needs to be added to the branching procedure to clean up these entries?
> 
> Yeah, I think that's where it got tricky for me to reason about. In theory, we'd update 1.12 and main with the change (and introduce a new changelog entry), but it seems like each branch should have its own changelog(-set) for Go versions.
> 
> I'll see if there's an outright disqualifier when running `changelog-build`. In the absence of that, we can leave the 1120 entry around until we decide on and document a process for this.

Okay, looks like `changelog-build` ignores _1120.txt, even if it's bumped after _1130.txt. So we can safely keep that file around for now.

**Review by cipherboy — APPROVED** [review]

**Comment by mpalmi** [comment]
We still need to bump the docker version in the docker race variant and probably add a note somewhere about docker versions.

File: website/README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ The website can be run locally through node.js or [Docker](https://www.docker.co
 
 ### With Docker
 
-Running the site locally is simple. Provided you have Docker installed, clone this repo, run `make`, and then visit `http://localhost:3000`.
+Running the site locally is simple. Provided you have the [latest version](https://docs.docker.com/engine/release-notes/) of Docker installed, clone this repo, run `make`, and then visit `http://localhost:3000`.
```
Comment (ncabatoff) [inline website/README.md:49]: Isn't this section just about building the website?  That's not impacted by the Go change is it?

Comment (mpalmi) [reply website/README.md:49]: I put this in here to keep our references to Docker installation consistent. If you think it doesn't belong, I can drop it, but I just saw it as a potential entrypoint to Vault + Docker.

The README recommends that you clone the repo and run `make` (no mention of "run make in the website dir"). If someone decided to then use that build with the older version of Docker and things went sideways, they might have some indication as to why.

Comment (ncabatoff) [reply website/README.md:49]: Fair enough.

**Pull request merged**